cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,25 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
```

Comment:
The class name 'Command' is too generic. It's better to give a more descriptive name that reflects its purpose, such as 'WaitForDbCommand'.